papers/helge_eichhorn/eichhorn.rst

```diff
@@ -0,0 +1,318 @@
+:author: Helge Eichhorn
+:email: eichhorn@dik.tu-darmstadt.de
+:institution: Technische Universität Darmstadt, Department of Computer Integrated Design
+
+:author: Reiner Anderl
+:email: anderl@dik.tu-darmstadt.de
+:institution: Technische Universität Darmstadt, Department of Computer Integrated Design
+
+--------------------------------------------------
+Plyades: A Python Library for Space Mission Design
+--------------------------------------------------
+
+.. class:: abstract
+
+    Designing a space mission is a computation-heavy task.
+    Software tools that conduct the necessary numerical simulations and optimizations are therefore indispensable.
+    The usability of existing software, written in Fortran and MATLAB, suffers because of high complexity, low levels of abstraction and out-dated programming practices.
+    We propose Python as a viable alternative for astrodynamics tools and demonstrate the proof-of-concept library Plyades which combines powerful features with Pythonic ease of use.
+
+.. class:: keywords
+
+   data modeling, object-oriented programming, orbital mechanics, astrodynamics
+
+Introduction
+------------
+
+Designing a space mission trajectory is a computation-heavy task.
+Software tools that conduct the necessary numerical simulations and optimizations are therefore indispensable and high numerical performance is required.
+Since no mission or spacecraft is alike the ever-changing requirements and constraints also demand high development speed and programmer productivity.
```

Comment:
That is what I wanted to say but the phrasing was rather clumsly. I reformulated it like so:

> No science mission or spacecraft are exactly the same and during the early mission design phases the technical capabilities and constraints change frequently.
> Therefore high development speed and programmer productivity are required as well.